app/subscribers/solidus_stripe/webhook/payment_intent_subscriber.rb

```diff
@@ -96,9 +94,15 @@ def complete_payment(payment)
       end
 
       def sync_refunds(event)
-        payment_method = event.spree_payment_method
-        payment_intent_id = event.data.object.id
+        event.data.object.to_hash => {
+          id: payment_intent_id,
```

Comment:
I'm not sure where the payment_intent_id is retrieved now 🤔

Reply:
How do you mean? :thinking:

Reply:
For a moment I got confused about how this block works
https://github.com/solidusio/solidus_stripe/blob/7bd100c7c0708e7ec82dc777fd4e930afc63f840/app/subscribers/solidus_stripe/webhook/payment_intent_subscriber.rb#L97-L102

😅  it's correct 🙏

Reply:
np! :slightly_smiling_face: